lms/djangoapps/verify_student/migrations/0005_auto__add_incoursereverificationconfiguration.py

```diff
@@ -97,7 +97,7 @@ def backwards(self, orm):
         },
         'verify_student.verificationcheckpoint': {
             'Meta': {'unique_together': "(('course_id', 'checkpoint_name'),)", 'object_name': 'VerificationCheckpoint'},
-            'checkpoint_name': ('django.db.models.fields.CharField', [], {'unique': 'True', 'max_length': '32'}),
```

Comment:
@awais786 after consulting with @wedaly and reviewing models I've determined this change does not require any action in the database.